### PR TITLE
Update to use latest Solidus v3.2

### PR DESCRIPTION
**Pull request by waiting-for-dev**
Also adds missing v3.1 to the older tests

### CODE REVIEW

File: src/commands/run-tests-solidus-older.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ description: >
 
 steps:
   - test-branch:
-      branch: v3.0
+      branch: v3.2
```
Comment (elia) [inline src/commands/run-tests-solidus-older.yml:6]: This should be already covered by `run-tests-solidus-current`

**Review by elia — CHANGES_REQUESTED** [review]
Looks good, just a small repetition in the commands

File: src/commands/run-tests-solidus-older.yml (hunk 1 above)
Comment (waiting-for-dev) [reply src/commands/run-tests-solidus-older.yml]: Oh, good catch, thank you. Fixed.

**Review by elia — APPROVED** [review]
Thanks!

**Pull request merged**